onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -147,25 +147,20 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level,
           converted_to_constant = false;
           break;
         }
-      }
 
-      if (converted_to_constant) {
-        for (size_t fetch_idx = 0; fetch_idx < fetches.size(); ++fetch_idx) {
```

Comment:
Previously we checked all the outputs were acceptable before adding new initializers. What's the reason for adding an initializer that may go unused if a later output value results in constant folding being stopped for this node?

Reply:
my bad... I changed this for some exp I was doing and missed to remove it